File: bluetooth/src/main/java/de/falkorichter/android/bluetooth/commandQueue/GattCommandQueueCallback.java
/**
 *
 */
package de.falkorichter.android.bluetooth.commandQueue;

import android.bluetooth.BluetoothGatt;

public interface GattCommandQueueCallback {
    public void finishProcessingCommands(NewGattCommandQueue queue, CALLBACKSTATE callbackstate, BluetoothGatt bluetoothGatt);

    public enum CALLBACKSTATE {
        FINISH_WITH_SUCCESS,
        FINISH_WITH_ERROR
    }
}

File: bluetooth/src/main/kotlin/de/falkorichter/android/bluetooth/commandQueue/GattCommandQueueCallback.kt
/**
 *
 */
package de.falkorichter.android.bluetooth.commandQueue

import android.bluetooth.BluetoothGatt

public trait GattCommandQueueCallback {
    public fun finishProcessingCommands(queue: NewGattCommandQueue, callbackstate: CALLBACKSTATE, bluetoothGatt: BluetoothGatt)

    public enum class CALLBACKSTATE {
        FINISH_WITH_SUCCESS
        FINISH_WITH_ERROR
    }
}
